### PR TITLE
Minimum Viable Governance

### PR DESCRIPTION
**Pull request by vrde**

### CODE REVIEW

File: DIPs/001/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+The proposal must contain:
+- The *problem* being addressed.
+- The *solution* proposed.
+- An *estimated effort*.
+- The *payment conditions*.
+
+The author of the proposal has the responsibility to gauge interest around their work, but it is not an obligation of the DAO to take their work into consideration.
+
+Being `n` the number of confirmations required to execute a transaction:
+- A proposal is **accepted** when at least `n` owners vote for it.
```
Comment (johba37) [inline DIPs/001/README.md]: how is "voting" happening?

File: DIPs/001/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Minimum Viable Governance
+This document describes the Minimum Viable Governance of Social Dist0rtion Protocol.
+
+The funds of the DAO are managed by the multi signature wallet deployed in the Main Ethereum Network at the address `0x527621278422FFf45A66f7086bAeAC770CF12b69`
+
+The multi signature wallet defines a *set of owners* and a *number of required confirmations* to execute a transaction.
```
Comment (johba37) [inline DIPs/001/README.md]: there are 2 possible threads imposed by the design of the gnosis multisig:
- if `number of required confirmations` == `number of owners`, then if one owner looses his key, the wallet is locked.
- if `number of required confirmations` much lower than `number of owners`, then a minority can collude and steal all funds.

ideas setup would be `number of required confirmations` = `number of owners - 1`. plus a rule to immediately report on lost keys.

File: DIPs/001/README.md (hunk 1 above)
Comment (vrde) [reply DIPs/001/README.md]: The voting to accept a proposal can be done offchain, to avoid calling a contract on-chain (this would be slow and cost money).

The voting can happen in the comments section of the issue. There should be a way to verify that the comment is coming from one of the owners. In this case we might need to implement a "sign and verify" page where a user can verify that the signatures are coming from the owners.

I'd love to have a simpler solution, if you have anything in mind please share :pray:

File: DIPs/001/README.md (hunk 2 above)
Comment (vrde) [reply DIPs/001/README.md]: Good point. I can make a proposal to increase the threshold from 2 to 3 on the multisig wallet. I've also added a paragraph about keys management.

File: DIPs/001/README.md (hunk 1 above)
Comment (johba37) [reply DIPs/001/README.md]: comment voting i think is great.

**Review by micheleb — APPROVED** [review]
looks good to me. If we're staying on github, we might write a simple bot to automate approval checks

File: DIPs/001/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The proposal may contain:
+- An *estimated effort*.
+- The *payment conditions*.
+
+The author of the proposal has the responsibility to gauge interest around their work, but it is not an obligation of the DAO to take their work into consideration.
+
+If the proposal is to be resolved on-chain (payments, minting, adding and removing owners, ...), owners may vote "yes" by confirming the proposed transaction in the multi signature wallet.
+
+Any other proposal can be voted by the owners in the comment section of the issue. If the number of "yes" votes is equal to the *number of required confirmations*, then the proposal is implemented.
```
Comment (sirnicolaz) [inline DIPs/001/README.md]: how is the number of required confirmations defined? Still majority based (everyone / 2 + 1)?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,14 @@
-# DAO
-Social DAOstortion Protocol
+# Introduction
+This repository is the home of the Social Dist0rtion Protocol (SDP) Decentralized Autonomous Organization (DAO).
+
+SDP's DAO is the combination of:
+- **Governance**, that describes the rules by which SDP operates. Those rules can be enforced by the members of SDP and/or by smart contracts.
+- A **multisig wallet**, to hold the funds of the DAO.
+
+SDP's governance is defined by a collection of standards called *Distortion Improvement Proposals* (DIPs).
```
Comment (sirnicolaz) [inline README.md]:
```suggestion
SDP's governance is defined by a collection of standards called *Dist0rtion Improvement Proposals* (DIPs).
```

**Review by sirnicolaz — CHANGES_REQUESTED** [review]
Just a little unclear how the number of required confirmations for the issue is defined. Not sure if it follows the majority rule mentioned above or other mechanisms. (see comment)

Plus there is a small typo in the spelling of Dist0rtion ;)

File: DIPs/001/README.md (hunk 3 above)
Comment (vrde) [reply DIPs/001/README.md]: It's defined in the multi signature wallet. I can rephrase it to:
> If the number of "yes" votes is equal to or greater than the *number of required confirmations* for the multisig wallet, then the proposal is implemented.

The threshold to transfer money is encoded in the *number of required confirmations* of the mutlisig wallet. In my opinion voting for other concerns should be handled with the same logic as executing transactions.

Comment (sirnicolaz) [reply DIPs/001/README.md]: then this formulation would make things easier to get for people like me (namely: n00bs)
```suggestion
Any other proposal can be voted by the owners in the comment section of the issue. If the number of "yes" votes is equal to the *number of required confirmations* defined in the multi signature wallet, then the proposal is implemented.
```

File: DIPs/001/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+### Improve it
+Anyone can open a pull request to this repository to submit a new DIP. If the pull request is approved by the majority of the owners, it is then merged to master, becoming part of the governance.
+
+## Funds
+The funds of the DAO are managed by the multi signature wallet deployed in the Main Ethereum Network at the address `0x527621278422FFf45A66f7086bAeAC770CF12b69`
+
+The multi signature wallet defines a *set of owners* and a *number of required confirmations* to execute transactions.
+
+## Proposals
+Anyone can make a **proposal** by opening an issue in any repository of the Social Dist0rtion Protocol organization on GitHub.
```
Comment (TimDaub) [inline DIPs/001/README.md:16]: I wonder if we should use the LeapDAO issue template.

I've been using it since two months now and I think it's really practical and easy to fill out.

This is the template:

```
# Bounty

## Scope
<!-- A list of specific things which should be done to deliver the bounty. These could be seen as requirements to verify/review bounty against -->

-
-
-

## Deliverables
<!-- Artifacts produced as the result of this bounty. Something that could be verified/reviewed. Some examples: updated code, deployment made, blog post published, public event conducted etc -->


## Gain for the project
<!-- How the completion of this bounty helps the project to achieve it is purpose -->


## Roles

bounty gardener: name / share
bounty worker: name / share
bounty reviewer: name / share

```

**Review by TimDaub — APPROVED** [review]
Great work 🎉

File: DIPs/001/README.md (hunk 5 above)
Comment (vrde) [reply DIPs/001/README.md:16]: What is the role of the gardener?

Comment (TimDaub) [reply DIPs/001/README.md:16]: //cc @johannbarbie (he knows better)

- Gardener: Creates the ticket, can answer questions to the worker, is requesting the payout from the multi signature
- Worker: Does the work
- Reviewer: Reviews the work

**Review by sirnicolaz — APPROVED** [review]

File: DIPs/001/README.md (hunk 5 above)
Comment (sirnicolaz) [reply DIPs/001/README.md:16]: I like also @vrde version though, it's quite simple, short and straight...

Comment (vrde) [reply DIPs/001/README.md:16]: I like the LeapDAO template and it can be added in another DIP to extend our governance. I'm reluctant to add it here because it would require also to explain the different roles and I want to keep it simple for now.

**Review by TimDaub — APPROVED** [review]

**Review by TimDaub — APPROVED** [review]
